### DISCUSSION TITLE
Installer is not working properly for ARO

### DISCUSSION THREAD

**Discussion by danielpenagos** [kubernetes-sigs/blob-csi-driver#996] [Q&A]
Any advice in regards executing the driver's installation process in Azure Red Hat OpenShift?. When executing the driver installation following the [guide](https://github.com/ezYakaEagle442/aro-pub-storage/blob/master/setup-store-CSI-driver-azure-blob.md#install-the-azure-blob-csi-driver) the  init container "install-blobfuse-proxy" finished with the following log:

```
/host/var/lib/kubelet/plugins/blob.csi.azure.com/blobfuse-proxy.sock
+ rm -rf /host/usr/bin/blobfuse-proxy
+ cp /blobfuse-proxy/blobfuse-proxy /host/usr/bin/blobfuse-proxy
cp: cannot create regular file '/host/usr/bin/blobfuse-proxy': Read-only file system
```
and the POD csi-blob-node-xxxxx is not able to start properly.

I guess this error is preventing the nginx-test pod to start: (To reach this stage I had to assign permissions to the cluster's Principal to the storage account (undocumented step))
```
MountVolume.MountDevice failed for volume "pvc-e3343d33-7eb0-4e40-8134-e5ffc1cec36a" : kubernetes.io/csi: attacher.MountDevice failed to create newCsiDriverClient: driver name blob.csi.azure.com not found in the list of registered CSI drivers
```

**Comment by andyzhangx** [comment]
blobfuse-proxy is not supported on ARO yet, can you specify `node.enableBlobfuseProxy=false` in helm chart install? thanks.
